### DISCUSSION TITLE
How do I make git only request passphrase once until the session is terminated

### DISCUSSION THREAD

**Discussion by thomasdkv** [community/community#70107] [New to GitHub]
Hi, I am currently using Windows 10 Pro, and I am pretty new to git/git bash. How do I make it so that I only have to type in the passphrase once at the first time pushing, and then I won't be prompted the passphrase request anymore whenever I do another push?

I followed the steps here, but it is still not working for me: https://docs.github.com/en/authentication/connecting-to-github-with-ssh/working-with-ssh-key-passphrases

Thanks in advance!

**Comment by queenofcorgis** [comment]
Hi there - are you getting an error message at all - if so which?
